packages/node-fetch/tests/URL.spec.ts

```diff
@@ -1,6 +1,36 @@
 import { PonyfillURL } from '../src/URL.js';
 
 describe('URL', () => {
+  const urlExamples = [
+    'https://example.com/?foo=bar&bar=qux',
+    'http://[::1]:8080',
+    'http://localhost:8080?foo=bar',
+  ];
+  for (const urlExample of urlExamples) {
+    describe(urlExample, () => {
+      it('should parse and stringify URLs', () => {
+        const url = new PonyfillURL(urlExample);
+        expect(url.toString()).toBe(urlExample.replace('8080', '8080/'));
+      });
+      it('should parse the URLs as expected', () => {
+        const ponyfillUrl = new PonyfillURL(urlExample);
+        const nativeUrl = new URL(urlExample);
+        expect(ponyfillUrl.toString() || null).toBe(nativeUrl.toString() || null);
+        expect(ponyfillUrl.protocol || null).toBe(nativeUrl.protocol || null);
```

Comment:
can results also be shared and not just the examples?

Reply:
What do you mean?

Reply:
ohhh I see now, nevermind, thanks!